README.md

```diff
@@ -95,4 +96,96 @@ If you highlight a specific service and press `enter` the highlighted will be se
 If you then press the `Create Transaction for Service` button, you will get a new menu to create a transaction for the selected service.
 
 ![Main tab of create keychain transaction](docs/img/main_keychain_transaction.png)
-The mechanism to configure a keychain transaction for a service is the same as the one for a transaction (described above). Only a few configuration are not possible (like the endpoint, the seed, the index and the elliptic curve).
\ No newline at end of file
+The mechanism to configure a keychain transaction for a service is the same as the one for a transaction (described above). Only a few configuration are not possible (like the endpoint, the seed, the index and the elliptic curve).
+
+# Archethic CLI
+It is also possible to call the archethic cli tool using the command line.
+
+## Generate address
+`generate-address`Get the address of a transaction based on parameters
+### Params
+- `--seed`  (string) the seed
+- `--index` (integer) index of the transaction
+- `--hash-algorithm`  (SHA256|SHA512|SHA3_256|SHA3_512|BLAKE2B) the hash algorithm. Default value is `SHA256`
+- `--elliptic-curve` (ED25519|P256|SECP256K1) the elliptic curve. The default value is `ED25519`
+
+## Send transaction
+`send-transaction`
+Send a transaction. It’s also possible to send a transaction for a specific service of a keychain (by passing the service parameter). You can either pass the parameter as flags in the command line or decide to put the parameters in a YAML file and pass the path of the config file using the `config` flag.
+### Command line params
+- `--config` (string) the path of the yaml configuration file (see below for the explanation of the parameters), if the config flag is passed, the other flags will be ignored for the configuration of the transaction
+- `--endpoint`  (local|testnet|mainnet|[custom url]) the endpoint to use, you can write your own URL. Default value is `local`.
+- `--access-seed`(string) the access seed
+- `--index` (integer) the index of the new transaction
+- `--elliptic-curve` (ED25519|P256|SECP256K1) the elliptic curve. The default value is `ED25519`
+- `transaction-type`  (keychain_access|keychain|transfer|hosting|token|data|contract|code_proposal|code_approval) the transaction type. The default value is `transfer`.
+- `--uco-transfers` (destinationAddress(string)=amount(integer)) the UCO transfers. You can create several UCO transfers in a transaction by passing the `uco-transfers` flag several times. The amount passed will be multiplied by 10^8.
```

Comment:
I propose `uco_transfer` and `token_transfer` as we can only add one transfer at the time by flag.